cli/src/cli.rs

```diff
@@ -27,33 +64,22 @@ pub enum Commands {
     Sync { commit: String },
     /// Print the information about the block data Git server that this node is hosting.
     Git,
-    /// Clean the repository, removing all the outdated (incompatible with `main`) branches.
+    /// Clean the repository, removing all the outdated (incompatible with `main`) commits.
```

Comment:
What's the difference and why? Is this what you intended at first but no one noticed it the last review?

Reply:
'commit's are more comprehensive than 'branches'. The initial one was wrong.

Reply:
Understood.